docs/amd_drug_metabolite.rst

```diff
@@ -0,0 +1,421 @@
+.. _amd_drug_metabolite:
+
+=====================
+AMD - Drug metabolite
+=====================
+
+Will create a drug metabolite model based on a model or dataset input. The given workflow is based on the given :ref:`strategy<strategy_amd>`.
+
+Regarding the dataset, two DVIDs are expected.
```

Comment:
No there are no checks on top level (maybe this is something we want?) but it will fail when running structsearch as adding a metabolite compartment assumes two DVIDs. Will change it to required nonetheless

Reply:
If it fails I think it should be caught at top level (not in this PR though).

Reply:
Of course, I'll open an issue for this